### ISSUE TITLE
Command Line is broken because ./vendor/bin/robo looks on wrong paths for autoload.php

### ISSUE THREAD

**Issue by tsmgeek** [consolidation/robo#1049]
Autoload paths are relative to either using phar or calling `robo` from the `vendor/consolidation/robo/robo`.

Add `__DIR__.'/../autoload.php'` to candidates in `robo`

https://github.com/consolidation/Robo/blob/3.x/robo#L19-L22

https://github.com/salesagility/SuiteCRM/issues/9224

**Comment by greg-1-anderson** [comment]
Cannot reproduce, a.k.a. "works on my machine".
```
$ mcd robovendortest
$ composer require consolidation/robo
$ ls -al vendor/bin/
total 0
drwxr-xr-x   5 ga  staff  160 Nov  5 15:49 .
drwxr-xr-x  11 ga  staff  352 Nov  5 15:49 ..
lrwxr-xr-x   1 ga  staff   44 Nov  5 15:49 release -> ../consolidation/self-update/scripts/release
lrwxr-xr-x   1 ga  staff   26 Nov  5 15:49 robo -> ../consolidation/robo/robo
lrwxr-xr-x   1 ga  staff   39 Nov  5 15:49 yaml-lint -> ../symfony/yaml/Resources/bin/yaml-lint
$ ./vendor/bin/robo --version
Robo 3.0.6
```
Is `vendor/bin/robo` a symlink in your use case? From the symptom, it sounds like Robo is running with the cwd set to `vendor/bin` instead of `vendor/consolidation/robo`.

I'll consider a PR to add the additional relative path, but it would be good to understand why it is needed.

**Comment by tsmgeek** [comment]
No it was not a symlink.
The app in question was running within a docker image and I've read that symlinks do not work within docker containers.
Either this or the deployment zip from suitecrm is not maintaining symlinks from their build process.
Adding extra path would account for situations where symlinks are lost depending on deployment/package choices.

**Comment by greg-1-anderson** [comment]
Makes sense. PRs welcome.

**Comment by tsmgeek** [comment]
Added PR

**Comment by greg-1-anderson** [comment]
Fixed in #1056
